src/components/prescriptions/PrescriptionComponent.vue

```diff
@@ -22,6 +22,10 @@
             type: String,
             default: 'Lekarz',
         },
+        date: {
+            type: String,
+            default: '10-01-2022',
```

Comment:
why default this date? suggest empty string or current date parse to appropriate format